certified-connectors/ShareEffect/apiDefinition.swagger.json

```diff
@@ -362,7 +386,102 @@
         "summary": "Get terms by label"
       }
     },
-    "/workflows/09a7432e83914ef0a36d3219b0a28a44/triggers/manual/paths/invoke": {}
+    "/workflows/09a7432e83914ef0a36d3219b0a28a44/triggers/manual/paths/invoke": {},
+    "/uploadtemplate": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
@Rens-Braspenning Default responses or Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger.